service/evidence/evidence_store.go

```diff
@@ -104,14 +105,15 @@
 		req, err = stream.Recv()
 
 		// If no more input of the stream is available, return
-		if err == io.EOF {
+		if errors.Is(err, io.EOF) {
```

Comment:
`errors.Is` is not necessary here, but also not wrong and good coding practice. According to DOC: "Read must return EOF itself, not an error wrapping EOF, because callers will test for EOF using =="

Reply:
Yes, but it *seems* that the error that is returned here is actually a wrapped gRPC error, so I chose the `errors.Is` approach.